consensus/ibft/signer/extra.go

```diff
@@ -37,6 +42,35 @@ type Seals interface {
 	UnmarshalRLPFrom(*fastrlp.Parser, *fastrlp.Value) error
 }
 
+// parseRoundNumber parses RLP-encoded bytes into round
+// FYI, Extra has 8 bytes space for round in order to distinguish between null and 0
+func parseRound(v *fastrlp.Value) (*uint64, error) {
+	roundBytes, err := v.Bytes()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(roundBytes) == 0 {
+		return nil, nil
+	}
+
+	if len(roundBytes) > 8 {
+		return nil, ErrRoundNumberOverflow
+	}
```

Comment:
Nitpick, maybe just reverse order of these two if statements, so that negative incomes are grouped together. WDYT?

```suggestion
	if len(roundBytes) > 8 {
		return nil, ErrRoundNumberOverflow
	}
	
	if len(roundBytes) == 0 {
		return nil, nil
	}
```

Reply:
Fixed c2fee48aa934d3210d2de82b9de7d339769ea35e